crates/runtime/src/actor/component/mod.rs

```diff
@@ -273,9 +273,7 @@ where
             | "wasi:logging/logging"
             | "wasi:sockets/tcp@0.2.0"
             | "wasmcloud:bus/lattice"
-            | "wasmcloud:bus/guest-config"
-            | "wasmcloud:messaging/messaging"
-            | "wasmcloud:messaging/message-subscriber" => continue,
```

Comment:
With this, we can remove the trait impl for `Messaging` for `Handler` in wasmbus I think

Reply:
Yeah it's on clear why these were left out, because the custom trait impl wasn't working either (and it couldn't even be targeted because of this being left out). Will investigate removing the trait impl!